extension/filterProductProperties.js

```diff
@@ -0,0 +1,22 @@
+module.exports = function (context, input, cb) {
```

Comment:
This extension it self should be used for any shop so white list are atm very customer specific.
We shouldnt put it in this extension.
I would just filter out the `Apply Map` property.

Reply:
The functionality of this script now only filters out the apply map property.